packages/xml-views-validation/src/utils/special-namespaces.ts

```diff
@@ -0,0 +1,4 @@
+export const TEMPLATING_NS =
```

Comment:
Should we have a map/dictionary instead of const per NS?

Reply:
I added the constants because they will be relevant in other validations too (like for the tag name)